update IPinfo vendor name to match context (#15613)
demisto/content: Packs/ipinfo/ReleaseNotes/2_1_0.md

#### Integrations
##### IPinfo v2

> - ***Breaking Change:*** Updated the **brand name** in indicators extracted and relationships, to match the context path `IPinfo`.

Reviewer: @amshamah419 wdyt?
@dorschw perhaps add the previous name so it'll be easier to keep track if anyone relied on the previous name.

> - ***Breaking Change:*** Updated the **brand name** in indicators extracted and relationships, to match the context path `IPinfo`.

Reviewer 2: ```suggestion
- ***Breaking Change:*** Updated the **brand name** for indicators extracted and their relationships, to match the context path `IPinfo`.
```
Does this explain what is going on?

> - ***Breaking Change:*** Updated the **brand name** for indicators extracted and their relationships, to match the context path `IPinfo`. (changing from  `ipinfo_v2`)

Reviewer: ```suggestion
- ***Breaking Change:*** Updated the **brand name** for indicators extracted and their relationships, to match the context path `IPinfo`(previously was `ipinfo_v2`). 
```
